apps/sms/views/inbox/js/inbox.js

```diff
@@ -114,14 +113,10 @@ var InboxView = {
     MessageManager.on('message-received', this.onMessageReceived.bind(this));
     MessageManager.on('threads-deleted', this.onThreadsDeleted.bind(this));
 
+    Drafts.init();
     Drafts.on('deleted', this.onDraftDeleted.bind(this));
     Drafts.on('saved', this.onDraftSaved.bind(this));
 
-    InterInstanceEventDispatcher.on(
-      'drafts-changed',
-      this.renderDrafts.bind(this, true /* force update */)
```

Comment:
Sounds good, and I found that we never have a test for the force === true actually... :p